sdk/tables/aztable/table_client_test.go

```diff
@@ -477,6 +478,43 @@ func (s *tableClientLiveTests) TestInvalidEntity() {
 	assert.Contains(err.Error(), partitionKeyRowKeyError.Error())
 }
 
+func (s *tableClientLiveTests) TestListEntities() {
+	require := require.New(s.T())
+	client, delete := s.init(true)
+	defer delete()
+
+	err := insertNEntities("partition", 5, client)
+	require.NoError(err)
+
+	count := 0
+	pager := client.List()
+	for pager.NextPage(context.Background()) {
+		resp := pager.PageResponse()
+		for _, e := range resp.TableEntityQueryResponse.Value {
+			require.Equal(e["PartitionKey"].(string), "partition")
+			count += 1
+		}
+	}
+	err = pager.Err()
+	require.NoError(err)
+	require.Equal(5, count)
+}
+
+func insertNEntities(pk string, n int, client *TableClient) error {
```

Comment:
nit: Should this go in `zt_table_recorded_tests` with all the other utility functions?